packaging/tags.py

```diff
@@ -407,7 +407,12 @@ def _mac_binary_formats(version, cpu_arch):
             return []
         formats.extend(["fat32", "fat"])
 
-    formats.append("universal")
+    if cpu_arch in {"arm64", "x86_64"}:
+        formats.append("universal2")
+
+    if cpu_arch in {"x86_64", "i386", "ppc64", "ppc"}:
```

Comment:
Looks like this should be:

```
     if cpu_arch in {"x86_64", "i386", "ppc64", "ppc", "intel"}:
```

Reply:
Okay, I can confirm that this change makes pip's test suite pass. Filing a PR for this now. :)